core/remote/remotemodelserver.cpp

```diff
@@ -294,10 +294,12 @@ bool RemoteModelServer::canSerialize(const QVariant &value) const
     // recurse into containers
     if (value.canConvert<QVariantList>()) {
         QSequentialIterable it = value.value<QSequentialIterable>();
-        for (const QVariant &v : it) {
-            if (!canSerialize(v))
-                return false;
+
+        bool cantSerialize = std::none_of(it.begin(), it.end(), [this](const QVariant &v) { return canSerialize(v); });
```

Comment:
actually, this is wrong now - my bad. make this:

```
const bool canSerialize = std::all_of(...);
if (!canSerialize)
```

Reply:
Hi @milianw I moved it back to use `any_of`, this will make it to return early since it does not needs to check if all elements can be serialized, if a single element is not, we can abort the checks and return false.